app/components/map.js

```diff
@@ -256,25 +220,26 @@ export class Map extends Component {
 
 
     render() {
-        let content;
-        if (this.props.locations && this.props.locations.length) {
-            content = getMapContent(this.props.showSingleTeam, this.props.locations);
-        } else {
-            content = [];
-        }
+        let content = () => {
+            if (this.props.locations && this.props.locations.length) {
+                return getMapContent(this.props);
+            } else {
+                return getMapContent(this.props);
+            }
+        };
 
         return (
             <View style={styles.container}>
                 <MapView
                     style={styles.container}
                     customMapStyle={mapStyle}
                     initialRegion={{
-                        latitude: 48.5690923,
-                        longitude: 7.6920547,
+                        latitude: 48.150676,
+                        longitude: 11.580984,
```

Comment:
Hahahaha What?!?!?! This was seriously hard coded?

Reply:
its the "region" of the map, I did center it the last time to be somewhere between Germany and Barcelona, not it focuses Germany